less/_grid.less

```diff
@@ -181,6 +181,37 @@
 				padding-bottom: 0;
 			}
 		}
+	}
+
+	// 3 COLUMNS
+	[class*='col'] {
+		@media @mq-md {
+			.sss&,
+			.ssw& {
+				width: 50%;
+				&:nth-child(3) {
+					width: 100%;
+				}
+			}
+			.sws& {
+				width: 100%;
+			}
+			.wss& {
+				width: 50%;
+				&:nth-child(1) {
+					width: 100%;
+				}
+			}
+		}
+	}
+
+	// 4 COLUMNS
+	[class*='col'] {
+		@media @mq-md {
+			.ssss& {
```

Comment:
This could be seen as overkill, but I wanted this to be an enhancement to the default layout, rather than the default styling.

This creates two rows of columns rather than four scrunched up columns or four stacked columns.